### PR TITLE
added item route to our app

### PR DESCRIPTION
**Pull request by tejasBhatGitHUb**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,4 +1,12 @@
 import uvicorn
+from fastapi import FastAPI
+from app.routes.item import route as item_route
+
+# setting up a Fastapi app
+app=FastAPI()
+
+# adding routes to our app
+app.include_router(router=item_route)
```
Comment (AIN2-00AL-G0WN) [inline main.py:9]: add prefix 'api' to item route.

Comment (AIN2-00AL-G0WN) [inline main.py:9]: add prefix 'api' to item route.

**Review by AIN2-00AL-G0WN — CHANGES_REQUESTED** [review]

File: main.py (hunk 1 above)
Comment (tejasBhatGitHUb) [reply main.py:9]: ok

**Review by AIN2-00AL-G0WN — APPROVED** [review]
now it looks good

**Pull request merged**